### PR TITLE
foo

### PR DESCRIPTION
**Pull request by vlerenc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

/hold
foo http://foo.sap

### CODE REVIEW

**Comment by gardener-botmanager** [comment]
@vlerenc Command /convert-to-draft not implemented yet. GitHub doesn't offer a callable mutation (see https://developer.github.com/v4/mutation) and has exposed this functionality only in the UI so far.

**Comment by gardener-botmanager** [comment]
@vlerenc Command "/review" failed with "ReadTimeout(ReadTimeoutError("HTTPSConnectionPool(host='api.github.com', port=443): Read timed out. (read timeout=15)"))".
<details>
<summary>Additional Information</summary>

```
Redacted in public. Check backend logs.
```
</details>

**Comment by gardener-botmanager** [comment]
@vlerenc Command `/draft` not implemented yet. GitHub doesn't offer a callable mutation (see [here](https://developer.github.com/v4/mutation), [here](https://github.community/t/no-v3-api-v4-graphql-mutation-to-convert-pr-to-a-draft/115699), and [here](https://github.com/github/hub/issues/1523#issuecomment-635341686)) and has exposed this functionality only in the UI so far.

**Comment by vlerenc** [comment]
dfgfdsgf

**Comment by vlerenc** [comment]
sdfdsf

**Comment by vlerenc** [comment]
No problem.

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,3 @@
+// Copyright
```
Comment (vlerenc) [inline CODEOWNERS:1]: /lgtm
foo http://foo.sap

Comment (vlerenc) [reply CODEOWNERS:1]: /hold

**Pull request closed without merging**